Pathogenesis proofreading (#1044)
greenelab/covid19-review: content/07.pathogenesis.md

Degrading or removing heparan sulfate was associated with decreased binding [@doi:10.1016/j.cell.2020.09.033].
Heparan sulfate may also interact with the S1/S2 proteolytic cleavage site and other binding sites to promote binding affinity [@doi:10.1101/2020.05.10.087288].
Notably, treatment with soluble heparan sulfate or even heparin (a commonly used anti-coagulant and vasodilator that is similar in structure to heparan sulfate [@doi:10.1007/978-3-642-23056-1_8]) potently blocked spike protein binding and viral infection [@doi:10.1016/j.cell.2020.09.033].
This finding is particularly interesting because degradation of heparan sulfate in the glycocalyx has previously been identified as an important contributor to ARDS and sepsis [@doi:10.1016/bs.ctm.2018.08.005], two common and severe outcomes of COVID-19, and suggests that heparan sulfate could be a target for pharmaceutical inhibition of cell entry by SARS-CoV-2 [@doi:10.1038/s41421-020-00222-5; @doi:10.1016/j.ebiom.2020.102969; @doi:10.1016/j.amjcard.2020.08.005; @doi:10.3390/nu13020328; @doi:10.3389/fimmu.2021.607306].

> Together, this evidence suggests that heparan sulfate can serve as an important adhesion molecule for SARS-CoV-2 cell entry and may represent a therapeutic target. <!-- It's slightly surprising to see the case made for heparan sulfate here because it doesn't come up at all (right?) in the therapeutics manuscript. Herapin is noted as SOC in an early trial, but that's all I recall. -->

Author: ```suggestion
Together, this evidence suggests that heparan sulfate can serve as an important adhesion molecule for SARS-CoV-2 cell entry.
It may represent a therapeutic target but has not been pursued as much as other candidate targets [@individual-pharmaceuticals].
```
How is this phrasing?  I think it is okay to point out the therapeutic hypothesis here, but I didn't want to suggest it is one of the leading candidates or even one that has been studied all that much.

> Together, this evidence suggests that heparan sulfate can serve as an important adhesion molecule for SARS-CoV-2 cell entry and may represent a therapeutic target. <!-- It's slightly surprising to see the case made for heparan sulfate here because it doesn't come up at all (right?) in the therapeutics manuscript. Herapin is noted as SOC in an early trial, but that's all I recall. -->

Reviewer: That is a great way to unify them!

In influenza A, for example, viral particles can be detected in aerosols produced by infected individuals, but it is not clear to what extent these particles drive the spread of influenza A infection [@pubmed:23771256; @doi:10.1097/QCO.0000000000000563; @doi:10.3201/eid1301.061202; @doi:10.1038/s41598-019-38825-y; @doi:10.1093/femsre/fuv039].
Regardless of its role in the spread of influenza A, however, aerosol transmission likely played a role in outbreaks such as the 1918 Spanish Influenza (H1N1) and 2009 "swine flu" (pH1N1) [@doi:10.1093/femsre/fuv039].
All three of these mechanisms have been identified as possible contributors to the transmission of HCoVs [@pubmed:23771256], including the highly pathogenic coronaviruses SARS-CoV-1 and MERS-CoV [@doi:10.1016/B978-0-12-385885-6.00009-2; @doi:10.1038/nrmicro.2016.81].

> Transmission of SARS-CoV-1 is thought to proceed primarily through droplet transmission, but aerosol transmission is also considered possible [@pubmed:23771256; @pubmed:15631748; @doi:10.1111/j.1600-0668.2004.00317.x; @doi:10.1086/429634], and fomite transmission may have also played an important role in some outbreaks [@doi:10.1371/journal.pone.0181558]. <!-- Were we able to get a translation of pubmed:15631748? -->

Reviewer: ```suggestion
Transmission of SARS-CoV-1 is thought to proceed primarily through droplet transmission, but aerosol transmission is also considered possible [@pubmed:23771256; @doi:10.1111/j.1600-0668.2004.00317.x; @doi:10.1086/429634], and as suggested by the English abstract of an article from China [@pubmed:15631748].
Fomite transmission may have also played an important role in some outbreaks [@doi:10.1371/journal.pone.0181558].
```
I don't think we were able to find anyone who could read it. I might have a solution to this longer term, but for now perhaps we should leave it like this. I think that's why I had phrased the language pretty weakly ("considered possible")

Author: In that case, how about we remove `@pubmed:15631748` from that list of references and keep the phrasing as-is?  We already have three others to support the statement.

I also added a citation to https://greenelab.github.io/covid19-review/#ref-sUAAY1me (d5ed3a09abc2d742796355239a26febbe4ef2443) in the later paragraph where we discuss SARS-CoV-2 specifically.  We had already cited in in pharmaceuticals but not here.

Reviewer: Thank you @agitter! This sounds good to me!

```suggestion
Transmission of SARS-CoV-1 is thought to proceed primarily through droplet transmission, but aerosol transmission is also considered possible [@pubmed:23771256; @doi:10.1111/j.1600-0668.2004.00317.x; @doi:10.1086/429634], and fomite transmission may have also played an important role in some outbreaks [@doi:10.1371/journal.pone.0181558]. <!-- Were we able to get a translation of pubmed:15631748? -->
```

Using their model, the authors also estimated the probability of new outbreaks occurring.
Assuming individual-level variation in transmission comparable to that of MERS or SARS, the probability of a single individual exporting the virus and causing a large outbreak is 17-25%, and assuming variation like that of SARS and transmission patterns like those observed for COVID-19 in Wuhan, the probability of a large outbreak occurring after ≥4 infections exist at a new location is greater than 50%.
An independent study came to similar conclusions, finding R~t~=2.38 in the two-week period before January 23 with a decrease to R~t~ = 1.34 (using data from January 24 to February 3) or R~t~=0.98 (using data from January 24 to February 8) [@doi:10.1126/science.abb3221].

> In South Korea, R~t~ was inferred for February through March 2020 in two cities, Daegu (the center of the outbreak) and Seoul [@tag:Park2020_distancing]. <!-- The original PDF on GitHub is gone, confirm the published version still supports these conclusions -->

Reviewer: ```suggestion
In South Korea, R~t~ was inferred for February through March 2020 in two cities, Daegu (the center of the outbreak) and Seoul [@tag:Park2020_distancing].
```
I confirmed that the journal version still shows the patterns described here!
